tests/rule-engine/bc-modus-ponens.scm

```diff
@@ -0,0 +1,56 @@
+; =============================================================================
+; Modus Ponens Rule
+;
+; Given P(A implies B) and sA, calculate sB
+; -----------------------------------------------------------------------------
+
+(define pln-rule-modus-ponens
+    (BindLink
+        (VariableList
+            (VariableNode "$A")
+            (VariableNode "$B"))
+        (ImplicationLink
+            (AndLink
+                (ImplicationLink
+                    (VariableNode "$A")
+                    (VariableNode "$B"))
+                (VariableNode "$A"))
+            (ExecutionOutputLink
+                (GroundedSchemaNode "scm: pln-formula-simple-modus-ponens")
+                (ListLink
+                    (VariableNode "$B")
+                    (ImplicationLink
+                        (VariableNode "$A")
+                        (VariableNode "$B")))))))
+
```

Comment:
Unit tests should be independent of the examples directory.  If someone decides to "improve" the examples, the unit tests should not start failing.  So William is doing the right thing.